libs/vkd3d/command.c

```diff
@@ -17279,6 +17330,12 @@ static void STDMETHODCALLTYPE d3d12_command_queue_ExecuteCommandLists(ID3D12Comm
         return;
     }
 
+    if (!(cmd_cost = vkd3d_calloc(num_command_buffers, sizeof(*cmd_cost))))
```

Comment:
missing free in fail path on line 17375.

Reply:
fixed.